chapters/annotations.tex

```diff
@@ -371,153 +365,77 @@ \section{Annotations for Code Generation}\label{annotations-for-code-generation}
 \end{semantics}
 \end{annotationdefinition}
 
-\begin{annotationdefinition}[HideResult]
-\begin{synopsis}[grammar]\begin{lstlisting}
-"HideResult" "=" ( false | true )
-\end{lstlisting}\end{synopsis}
-\begin{semantics}
-\lstinline!HideResult = true! defines that the model developer proposes to not show the simulator results of the corresponding component.
 
-\lstinline!HideResult = false! defines that the developer proposes to show the corresponding component.
-
-\begin{nonnormative}
-For example, a tool is not expected to provide means to plot a variable with \lstinline!HideResult = true!.  If a variable is declared in a protected section, a tool might not include it in a simulation result. By setting \lstinline!HideResult = false!, the modeler would like to have the variable in the simulation result, even if in the protected section.
+\section{Annotations for Simulations}\label{annotations-for-simulations}
 
-\lstinline!HideResult! is for example used in the connectors of the \lstinline!Modelica.StateGraph! library to not show variables to the modeler that are of no interest to him and would confuse him.
-\end{nonnormative}
-\end{semantics}
-\end{annotationdefinition}
+The annotations listed below define how models can be checked, translated, and simulated.
+\begin{center}
+\begin{tabular}{l|l l}
+\hline
+\tablehead{Annotation} & \tablehead{Description} & \tablehead{Details}\\
+\hline
+\hline
+\lstinline!experiment! & Simulation experiment settings & \Cref{modelica:experiment}\\
+\lstinline!HideResult! & Don't show component's simulator result & \Cref{modelica:HideResult}\\
+\lstinline!TestCase! & Information for model used as test case & \Cref{modelica:TestCase}\\
+\hline
+\end{tabular}
+\end{center}
 
-\begin{annotationdefinition}[Inline]
+\begin{annotationdefinition}[experiment]
+% henrikt-ma 2021-04: Seems strange to allow 'experiment' completely without list of options -- what would it mean?
```

Comment:
Should we open a ticket about it?

Reply:
Good suggestion, but it seems like this comment was lost when the PR was closed.  Opened #2985.